### PR TITLE
Add ec2_key_info module

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

Separated the info module from PR #1704 

##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/ec2_key_info.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            description: The public key material.
+            returned: always
+            type: str
+        create_time:
+            description: The time the key pair was created.
+            returned: always
+            type: str
+            sample: "2023-08-16T10:13:33.025000+00:00"
+        tags:
+            description: a dictionary representing the tags attached to the key pair
```
Comment (alinabuzachis) [inline plugins/modules/ec2_key_info.py:105]:
```suggestion
            description: A dictionary representing the tags attached to the key pair.
```

File: changelogs/fragments/add_ec2_keypair_info.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply changelogs/fragments/add_ec2_keypair_info.yml:2]: The new module will be automatically added to the changelog entries when the release changelog will be generated, that's why we do not need the changelog fragment.

[the remark this answers is not in the document]
Comment (hakbailey) [reply changelogs/fragments/add_ec2_keypair_info.yml]: Can the changes be put in a `trivial` block so it passes CI?

**Review by alinabuzachis — APPROVED** [review]

File: changelogs/fragments/add_ec2_keypair_info.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (GomathiselviS) [reply changelogs/fragments/add_ec2_keypair_info.yml]: @alinabuzachis added the skip-changelog label, which should skip the test.

**Review by gravesm — APPROVED** [review]

**Pull request merged**